src/QMCDrivers/WaveFunctionTester.cpp

```diff
@@ -69,6 +70,12 @@ WaveFunctionTester::WaveFunctionTester(const ProjectData& project_data,
 
   deltaR.resize(w.getTotalNum());
   makeGaussRandom(deltaR);
```

Comment:
Any reason not to make makeGaussRandom dimension aware and avoid the changes and logging on later lines?

Reply:
The reason is that I wanted the change to be localized in the WaveFunctionTester.

Eventually, what you suggested is nicer. Although, I would have to touch a lot of files to make this API change.
I wonder if it's worth making another PR for that.

Reply:
Hi Paul - In that case I agree that making another PR would be a good way of splitting this up